code/modules/cargo/exports/seeds.dm

```diff
@@ -26,12 +26,12 @@
 	export_types = list(/obj/item/seeds)
 	needs_discovery = TRUE // Only for already discovered species
 
-/datum/export/seed/potency.get_cost(obj/O)
+/datum/export/seed/potency/get_cost(obj/O)
```

Comment:
DM!!
DM WHYYY!!

Reply:
what the absolute fuck

Reply:
my favorite part of this PR is remie explaining (accurately, I might add) why two of the quirkiest things here are working, but just giving up when she sees this bullshit

Reply:
don't forget the one below, a super proc call IN A DEFINITION???

Reply:
fun fact: it's actually the header to an empty tree block for `..`, i.e. one level up. The `()` are just ignored.

Reply:
I figured it was a path operator thing, but it doesn't make it any less dumb.

Reply:
worse, relatively pathed path operators. *shudder*

Reply:
dot path operators have to be among the most frightening and arcane byond features we know of

Reply:
You mean the thing we use in every callback honk

Reply:
they make sense for callbacks! that's like, an actually useful case for them
it's stuff like `......` that really gets me
did we ever figure out why that one works when `.....` doesn't?

Reply:
as the discoverer of ```.....``` (5dot, 6dot actually doesn't work except in REALLY arcane contexts where 5dot doesn't work, at which point 6dot starts acting like 5dot) I don't consider it a path operator.
It happens IN the proc, without the need for any paths around it, it's just a really arcane sort of lookup.

if I were to swap ```.``` the variable for the word ```DOT```, I would think it comes down to this:
```DOT.DOT.DOT```

Which, I guess makes sense? it's looking up DOT's DOT's DOT variable, but as for why 5dot stops working in some contexts, and 6dot (which shouldn't work??? ```DOT.DOT.DOT.```) works instead, I haven't the foggiest.

Reply:
Now its on my mind, I'm going to ask lummox.

Reply:
@AnturK @duncathan @MrStonedOne 
http://www.byond.com/forum/?post=2311084

support my post you nerds.